### PR TITLE
rework UI for coin control

### PR DESCRIPTION
**Pull request by valpinkman**
### Type

UI Improvement

### Context

LL-1684

### Parts of the app affected / Test plan

Coin Control
image
Screenshot 2020-08-21 at 16 19 34
image

image

### CODE REVIEW

**Review by dasilvarosa — CHANGES_REQUESTED** [review]
Awesome work! Some minor remarks:

**Send step 2 (Amount)**
- [x] Change order of the toggles: RBF first.
- [ ] RBF enabled by default.
- [ ] Reword (add a comma): “Include coins from unconfirmed, replaceable transactions”
- [ ] Reword: "Allow speeding up the transaction later by increasing the network fees (Replace-By-Fee)"

**Select coins modal**
- [ ] Reword title: Select coins --> Coin control
- [ ] DOGE 10 —> 10 DOGE to be consistent with the new way of displaying units
- [ ] More spacing between CTA and support link
- [ ] Reword strategies: “Oldest coins first (FIFO)“, “Minimize fees (optimize size)“, “Minimize future fees (merge coins)” 
- [ ] Slight reword:  "You cannot select coins from unconfirmed, replaceable transactions."

**Comment by valpinkman** [comment]
>  Implement not enough coins selected error case like in the design. Make sure it's stickied when scrolling down the list of inputs.
@dasilvarosa seen with Chris, we keep the same banner for now, as there will be a new task to update them.
- [x] sticky done
I believe if we want a new error text, we need to throw a new error type (this involves libcore I think)

**Review by dasilvarosa — APPROVED** [review]

**Review by Arnaud97234 — CHANGES_REQUESTED** [review]
### Bugs found

1) In the coin control modal -> Pending tx: "Pending" and "Replaceable" text should be in Red.

Actual
actual

Expected
expected
___________
___________
2) UTXOs shouldn't be displayed twice

Screenshot 2020-08-31 at 19 39 47
___________
___________

3) Maximum spendable should adapt when user turns OFF "Include coin from unconfirmed / replaceable tx" AND there are some UTXOs corresponding to RBF tx.
- if OFF -> Max spendable = Balance - Replaceable UTXOs

ezgif com-video-to-gif

**Review by Arnaud97234 — CHANGES_REQUESTED** [review]
In the coin control modal -> Pending tx: "Pending" and "Replaceable" text should be in Red.

Actual
actual

Expected
expected

**Review by Arnaud97234 — DISMISSED** [review]

**Review by dasilvarosa — APPROVED** [review]

File: src/renderer/components/OperationsList/AddressCell.js [hunk 1]
```diff
@@ -25,6 +25,25 @@ const Address = ({ value }: { value: string }) => {
   );
 };
 
+export const SplitAddress = ({ value }: { value: string }) => {
```
[the remark this answers is not in the document]
Comment (valpinkman) [reply src/renderer/components/OperationsList/AddressCell.js]: I changed this, reusing the same component.

**Review by IAmMorrow — DISMISSED** [review]

**Review by Arnaud97234 — DISMISSED** [review]

**Review by Arnaud97234 — APPROVED** [review]

**Pull request merged**